benchmark/python/benchmark_hail/run/table_benchmarks.py

```diff
@@ -333,3 +333,9 @@ def join_p100_p10():
 def group_by_collect_per_row():
     ht = hl.read_matrix_table(resource('gnomad_dp_simulation.mt')).localize_entries('e', 'c')
     ht.group_by(*ht.key).aggregate(value=hl.agg.collect(ht.row_value))._force_count()
+    
+
+@benchmark
+def group_by_take_rekey():
```

Comment:
can we add matrix table group_rows_by benchmarks that are the same as these two?

Reply:
yeah---do you want me to do that here? I think they *should* lower into the same thing, right? so that would mostly serve to check that we're not messing up the lowering?

Reply:
don't need to do it here, but it shouldn't be hard. The matrix one will be an array agg on the entries, rather than the whole row.